test/testtokenize.cpp

```diff
@@ -8492,6 +8493,12 @@ class TestTokenizer : public TestFixture {
         ASSERT_NO_THROW(tokenizeAndStringify(code));
     }
 
+    void stringviewLiteral() { // #13234
+        const char code[] = "auto sv2 = \"test\"sv;";
```

Comment:
I fear this does not match the test code in the ticket. I fear the user included string_view and that his header is needed to reproduce the syntax error.